finrl/env/env_stocktrading_cashpenalty.py

```diff
@@ -247,27 +261,36 @@ def step(self, actions):
             self.account_information["reward"].append(reward)
 
             # multiply action values by our scalar multiplier and save
-            actions = actions * self.hmax
+            actions = actions * self.hmax 
+            #buy/sell only if the price is > 0 (no missing data in this particular date 
+            actions = np.array([actions[i] if closings[i]>0 else 0 for i in range(len(actions))])
```

Comment:
there's a numpy way to do this, let's do that instead of the list comp. 
I believe it might be np.where

Reply:
Sure